src/SamplesApp/UITests.Shared/Windows_UI_Xaml/DragAndDrop/DragDrop_Files.xaml.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using Windows.ApplicationModel.DataTransfer;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+using Uno.UI.Samples.Controls;
+
+#if HAS_UNO_WINUI || WINAPPSDK
+using Microsoft.UI.Input;
+#else
+using Windows.Devices.Input;
+using Windows.UI.Input;
+#endif
+
+namespace UITests.Windows_UI_Xaml.DragAndDrop
+{
+	[Sample(
+		Description =
+			"This test showcases the common scenario of dragging files and/or folders from another window (usually a file explorer) into a specific rectangle that accepts the drop. The files that were dragged and dropped should be seen as a list.",
+		IgnoreInSnapshotTests = true)]
```

Comment:
@ramezgerges Let's mark as manual test, so QA can see it in the samples app. Same for the other clipboard sample.